### ISSUE TITLE
Typos in descriptions

### ISSUE THREAD

**Issue by GameSoul7Eugene** [Scirra/Construct-3-bugs#6204]
**1.** ![orthographic1](https://user-images.githubusercontent.com/59319082/196968217-1c87ebe1-72e6-4b50-bda8-abba9eac5648.png)
**2.** ![orthographic2](https://user-images.githubusercontent.com/59319082/196968246-8876d055-a76a-42a3-9a41-cd117a3825c4.png)
**3.** ![hashtag1](https://user-images.githubusercontent.com/59319082/196968263-4d122bae-1ee9-4ca4-bf81-b8145560fc34.png)
**4.** ![hashtag2](https://user-images.githubusercontent.com/59319082/196968281-415d6118-fb26-4fb6-9e8d-fdefae33c165.png)
**5.** ![other1](https://user-images.githubusercontent.com/59319082/196968303-8328a227-1460-47c9-bc0f-2a026e6fc568.png)
**6.** ![other2](https://user-images.githubusercontent.com/59319082/196968326-9c43f931-6c7b-4a7c-a893-4b5771f6fdd9.png)
**7.** ![other3](https://user-images.githubusercontent.com/59319082/196968342-8cf2fc5a-504f-4f6d-8372-d2462c270071.png)
**8.** ![other4](https://user-images.githubusercontent.com/59319082/196968364-6761b854-b0a6-4b9a-ae01-e257418ca975.png)
**9.** ![other5](https://user-images.githubusercontent.com/59319082/196968403-531f1196-e95d-4a82-8e1f-a90652edc9a1.png)
**10.** ![other6](https://user-images.githubusercontent.com/59319082/196968432-0b7fbb9f-9e8c-4b36-9353-570044f41bc4.png)
**11.** ![other7](https://user-images.githubusercontent.com/59319082/196968458-94a55d28-15bf-466d-a931-9579be9c636b.png)
**12.** ![other8](https://user-images.githubusercontent.com/59319082/196968486-4e6836c1-dffa-4ba4-84e4-f749a84c687a.png)
**13.** ![other9](https://user-images.githubusercontent.com/59319082/196968512-13f1841a-e103-462d-b13a-9c1fc80b4ee6.png)
**14.** ![other10](https://user-images.githubusercontent.com/59319082/196968537-d366a947-1cf2-4f5b-8378-bed2670b6c90.png)
**15.** ![other11](https://user-images.githubusercontent.com/59319082/196968558-7708e78e-1edc-4d5f-9738-caf2d7370a44.png)
**16.** ![other12](https://user-images.githubusercontent.com/59319082/196968575-a053942d-e246-4a18-a90b-28228b533947.png)
**17.** ![other13](https://user-images.githubusercontent.com/59319082/196968594-c237a885-0652-4a6e-9e6a-2d3d15803646.png)
**18.** ![other14](https://user-images.githubusercontent.com/59319082/196968628-1a1c50b7-2108-4023-84d4-b966992263cc.png)
**19.** ![other15](https://user-images.githubusercontent.com/59319082/196968671-003e7215-8993-44f2-bfbe-592723928f34.png)
**20.** ![other16](https://user-images.githubusercontent.com/59319082/196968692-00cc04eb-314b-4103-bbc4-3379dc72a249.png)
**21.** ![other17](https://user-images.githubusercontent.com/59319082/196968723-94e7f226-61d1-47d5-a2f8-5ed7c8a263e4.png)

**Comment by brushfe** [comment]
Great catches! 

I apologize for jumping on your bandwagon, but it's related—TileMovement uses "grid dimensions" and "grid size" for the same action:

![Screen Shot 2022-10-21 at 6 23 07 PM](https://user-images.githubusercontent.com/63005234/197309010-846333a8-64b0-44f0-8e21-e71ecf01e66f.png)

**Comment by AshleyScirra** [comment]
Well spotted, thanks for listing these. I've updated them all, but to avoid invalidating lots of translations shortly before a stable release, these changes will go through for the next release cycle.

@brushfe - I did not actually update the one you mentioned as "grid dimensions" is referring to both the size and the offset - the "grid size" references only the size parameters and is followed by the offset parameters which are mentioned separately, and using the action name "Set grid size" would omit the fact it also changes the offset.
